config/locales/en/views.yml

```diff
@@ -195,8 +195,11 @@ en:
         in the box below.
       feedback_form: Feedback form
       body: Your message
-      body_hint: >-
-        Do not include sensitive personal information, such as prisoner numbers.
+      date_of_birth: Prisoner date of birth
```

Comment:
Due to the way the partial works we need them here - it derives the scope from the object name.